pithos/pithos.py

```diff
@@ -669,7 +672,19 @@ def current_song(self):
         if self.current_song_index is not None:
             return self.songs_model[self.current_song_index][0]
 
+    def register_if_ad(self, song_index):
+        #If the upcoming song is an ad, register_ad before we possibly get a new playlist.
+        #The theory is that we want Pandora to know we heard any ads in the last playlist
+        #before we ask for another one so hopefully they send us as few ads as possible.
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->